### PR TITLE
Fix handbook linking 0.2.0 version to 0.3.0

### PR DESCRIPTION
**Pull request by Jakio815**
Fix https://www.lf-lang.org/docs/handbook/command-line-tools?target=c page linking 0.2.0 to 0.3.0.

### CODE REVIEW

**Comment by Jakio815** [comment]
@lhstrh Not sure if the review is assigned. Just leaving a note in case of no notifications.

File: packages/documentation/copy/en/reference/Target Declaration.md [hunk 1]
```diff
@@ -337,7 +337,7 @@ target Cpp {
 };
 ```
 
-This will optionally append additional custom CMake instructions to the generated `CMakeLists.txt`, drawing these instructions from the specified text files (e.g, `foo.txt`). The specified files are resolved using the same file search algorithm as used for the [files](#files) target parameter. Those files will be copied into the `src-gen` directory that contains the generated sources. This is done to make the generated code more portable<span class="lf-c"> (a feature that is useful in [federated execution](/docs/handbook/distributed-execution)</span>.
+This will optionally append additional custom CMake instructions to the generated `CMakeLists.txt`, drawing these instructions from the specified text files (e.g, `foo.txt`). The specified files are resolved using the same file search algorithm as used for the [files](#files) target parameter. Those files will be copied into the `src-gen` directory that contains the generated sources. This is done to make the generated code more portable<span class="lf-c"> (a feature that is useful in [federated execution](/docs/handbook/distributed-execution))</span>.
```
Comment (Jakio815) [inline packages/documentation/copy/en/reference/Target Declaration.md:340]: Added missing bracket.

**Comment by Jakio815** [comment]
@edwardalee Would you please review this? Just fixed some minor typos while reading the handbook.

**Comment by edwardalee** [comment]
LGTM! Thanks.

**Pull request merged**